src/scratchOrgSettingsGenerator.ts

```diff
@@ -62,7 +78,10 @@ export default class SettingsGenerator {
   private objectSettingsData?: { [objectName: string]: ObjectSetting };
   private logger: Logger;
   private writer: ZipWriter;
+  private allRecTypes: string[] = [];
+  private allBps: string[] = [];
```

Comment:
Could you expand these out to the full words instead of using abbreviations please?

It's hard to understand what `RecTypes` and `Bps` means if you aren't super familiar with the codebase.